README.md

```diff
@@ -427,6 +427,27 @@ from your `lein repl` console output.
 Finally, use `M-x ejc-connect` from any SQL buffer to connect to the exact database,
 as always.
 
+### Mix ejc-sql with Org Mode default ob-sql engine together in one buffer
+
+You can set option `ejc-org-mode-babel-wrapper` to disable `ejc-sql` wrapper
```

Comment:
To disable `ejc-sql` wrapper around Org Mode SQL source code blocks, set `ejc-org-mode-babel-wrapper` to `nil` (enabled by default).